apptest/deployer/schema.yaml

```diff
@@ -64,17 +15,5 @@ properties:
 
   conjur-oss.ssl.hostname:
     type: string
-    default: "conjurtest.myorg.com"
+    default: "$CERTIFICATE_CN"
     title: Certificate Common Name
-
-required:
-- name
-- namespace
-- conjur.image
-- postgres.image
-- nginx.image
-- conjur-oss.dataKey
-- conjur-oss.persistentVolumeSize
-- conjur-oss.ssl.expiration
-- conjur-oss.ssl.hostname
-- conjur-oss.serviceAccount.name
```

Comment:
Are these no longer required? Seems odd that they would just be removed.

Reply:
From what I've seen, the `apptest/deployer/schema.yaml` gets merged with the project `schema.yaml` for testing/verification... so the `apptest/deployer/schema.yaml` should only need the incremental properties for testing. I base this understanding on this documentation:
https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/master/docs/verification-integration.md#2-writing-the-test-schemayaml

Also, here's a WordPress example from Google that serves as a baseline for writing a Google Marketplace app:
https://github.com/GoogleCloudPlatform/marketplace-k8s-app-example/blob/master/wordpress/apptest/deployer/schema.yaml

And the corresponding project schema.yaml for the WordPress app:
https://github.com/GoogleCloudPlatform/marketplace-k8s-app-example/blob/master/wordpress/schema.yaml

I added a comment explaining this at the top of this schema.yaml file to make this clear.